### DISCUSSION TITLE
Error while running for a sitemap.xml

### DISCUSSION THREAD

**Discussion by dhineshbabu95** [GoogleChromeLabs/ps-analysis-tool#868] [Support Forum] [answered]
o: Network.enable timed out. Increase the 'protocolTimeout' setting in launch/connect calls for a higher timeout if needed.

error when I ran for a sitemap.xml file with 567 pages it failed somewhere around 250 pages in.

how can I bypass this?

**Comment by milindmore22** [comment]
Hello @dhineshbabu95, 

Thank you for reaching out to us. We understand that you are encountering difficulties running analysis for sitemap. We have attempted to analyze sitemaps of substantial websites without encountering similar issues. To assist you more effectively, **please provide the specific sitemap URL or file that is causing the problem.**

Additionally, kindly confirm that you are utilizing the latest version of the PSAT CLI (**0.11.0-1**). To verify this, please execute the following command, depending on your installation method:

Cloned and built from repository: 
```shell
npm run cli -- -V
```
npm package:
 
```shell
psat -V
```

If you are not on the latest version, please make sure to update with the help of our [wiki guide](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface#installing-psat-cli-as-package)

**Comment by dhineshbabu95** [comment]
Hi @milindmore22

I do have the updated version
<img width="289" alt="image" src="https://github.com/user-attachments/assets/6187fb3b-ab00-459d-96ac-1b471baa6e03">

I ran the command `npm run cli -- -s https://www.lexus.com.au/sitemap.xml`

**Comment by milindmore22** [comment]
Hello @dhineshbabu95,

Thank you for your confirmation the PSAT CLI version.

We are able to execute the same command without encountering any difficulties. (attached zip of report generated)
[www-lexus-com-au-sitemap-xml.zip](https://github.com/user-attachments/files/17748391/www-lexus-com-au-sitemap-xml.zip)

The error message indicates a potential timeout issue during the connection phase, which may be attributed to network instability, firewall restrictions, or proxy server configuration.

**Comment by dhineshbabu95** [comment]
Thanks for your prompt response @milindmore22.
 
[PSAT error.txt](https://github.com/user-attachments/files/17942543/PSAT.error.txt)
This the error I get. Is there any variable I can change to bypass this? Cause I'd like to extract it myself.

**Comment by milindmore22** [comment, marked as answer]
Hello @dhineshbabu95 

Thanks for sending the error report! We tried to recreate the issue, and noticed you're using Node.js v20.11.1. While it works for some things, some packages needed for PSAT might not be compatible with this version.

We tested with the latest Node.js version (v20.18.1) and everything worked smoothly when running PSAT CLI commands.

To ensure a smooth experience and avoid potential errors, we recommend using the Node.js version specified in the project's [.nvmrc ](https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/main/.nvmrc) file ie Node.js version v18

To switch between node version I will recommend using [nvm](https://github.com/nvm-sh/nvm)
